single_tech_samples/databricks/sample4_ci_cd/pyspark/devops/ci-pipeline.yml

```diff
@@ -0,0 +1,73 @@
+# Starter pipeline
```

Comment:
I'm getting this warning when trying to run this pipeline:
![image](https://user-images.githubusercontent.com/5771063/119078487-a3dcf080-ba39-11eb-894c-e1d0f345bcc0.png)

Reply:
Commit **[0984ae2](https://github.com/Azure-Samples/modern-data-warehouse-dataops/pull/345/commits/0984ae2777a05c5d42c6a44da9b57feefda6db75)** has fixed this warning issue.
Thanks for pointing this out! :-)